Solution.yaml

```diff
@@ -95,6 +95,18 @@ parameters:
       tooltipText:
         fr: 'Choisissez une date de début'
         en: 'Choose a start date'
+      validation: '< end_date'
+  - id: 'end-date'
```

Comment:
Shouldn't it be `end_date` instead of `end-date`?

Reply:
oh :facepalm: thanks, I fix it!